### ISSUE TITLE
v0.1.13 bug

### ISSUE THREAD

**Issue by aljazceru** [bitvora/wot-relay#79]
tried running v0.1.13 or latest master and both lead to the same result:
```
./wot-relay 
	
2024/11/18 17:04:11 🚀 booting up web of trust relay
2024/11/18 17:04:11 🔄 refresh interval set to 1 hours
badger 2024/11/18 17:04:11 INFO: All 0 tables opened in 0s
badger 2024/11/18 17:04:11 INFO: Discard stats nextEmptySlot: 0
badger 2024/11/18 17:04:11 INFO: Set nextTxnTs to 0
2024/11/18 17:04:11 🎉 relay running on port :3334
2024/11/18 17:04:11 🔍 fetching owner's follows
2024/11/18 17:04:12 🌐 building web of trust graph
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x99d365]

goroutine 222 [running]:
github.com/dgraph-io/badger/v4.(*Iterator).Next(0xc0068ee6e0)
	/root/go/pkg/mod/github.com/dgraph-io/badger/v4@v4.2.0/iterator.go:586 +0x25
github.com/fiatjaf/eventstore/badger.(*BadgerBackend).QueryEvents.func1(0xc005a52800)
	/root/go/pkg/mod/github.com/fiatjaf/eventstore@v0.11.2/badger/query.go:120 +0x92a
github.com/dgraph-io/badger/v4.(*DB).View(0xc005ec9b28?, 0xc005a41600)
	/root/go/pkg/mod/github.com/dgraph-io/badger/v4@v4.2.0/txn.go:807 +0x8f
created by github.com/fiatjaf/eventstore/badger.(*BadgerBackend).QueryEvents in goroutine 53
	/root/go/pkg/mod/github.com/fiatjaf/eventstore@v0.11.2/badger/query.go:45 +0x20b
```

**Comment by cr0bar** [comment]
Also receiving this error on Debian Linux with latest Go. Rolled back to older build.

**Comment by fiatjaf** [comment]
I believe this was fixed in eventstore v0.11.3.

**Comment by barrydeen** [comment]
@cr0bar fixed now in latest https://github.com/bitvora/wot-relay/releases/tag/v0.1.14
